creativecommons/legaldb
# [Feature] Add Open Graph attributes for the cc legal db page

## Fixes



- Fixes #253  by @SylviaUzoh1

## Description

This PR adds the appropriate Open graph meta tags for the legal db project.

## Technical details

This PR includes the following meta tags;
- `og:url`
- `og:title`
- `og:type`
- `og:description`
- `og:image`
- `og:locale`
- `og:locale:alternate`

## Tests

- Run the development server on your machine
- Install the localhost Open graph checker on your browser (chrome)
- Open the devlopment instance of the project and click the extension
- Copy the generate url and test on both facebook and x to confirm the appropriate og meta content is being displayed.


## Screenshots

![image](https://github.com/user-attachments/assets/19e228af-1357-4b74-ac4f-f1718848c5e5)
<img width="572" alt="Screenshot 2024-10-31 at 20 34 46" src="https://github.com/user-attachments/assets/9da7aa32-279a-4a9c-a878-a124d842493a">



## Additional Information
Link to the official plugin page: [Localhost Open Graph Checker - Chrome Web Store](https://chromewebstore.google.com/detail/localhost-open-graph-chec/gcbnmkhkglonipggglncobhklaegphgn?hl=en)

## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
